### ISSUE TITLE
bug: Ionic/React - I can only get one slice of Toast

### ISSUE THREAD

**Issue by burkmarr** [ionic-team/ionic#19823]
**Ionic version:**
ionic/react 4.11.3

**Current behavior:**
Setting the isOpen attribute on an <IonToast> element to true shows the toast. But once it is dismissed, setting the attribute to true again does not cause the toast element to reappear. 

**Expected behavior:**
The toast element should reappear when the isOpen attribute is set to true.

**Steps to reproduce:**
Deploy the React code example code on the Ionic Framework API page: https://ionicframework.com/docs/api/toast

Click the button 'Show Toast 1' or 'Show Toast 2'. Wait for the toast to dissappear or dismiss it. Click the same button again - the toast does not reappear.

**Ionic info:**
```
Ionic:

   Ionic CLI : 5.4.5

Utility:

   cordova-res : not installed
   native-run  : not installed

System:

   NodeJS : v10.15.3
   npm    : 6.4.1
   OS     : Windows Server 2016
```

**Comment by theknicker** [comment]
I am also having nearly the same issue, except each subsequent time the toast is opened, it shows up for less and less time.

I have the duration set to 1500, yet the second time I open the toast it only stays open for a split second, not even long enough to read the message. After 3 or 4 times, it stops opening up completely.

**Comment by elylucas** [comment]
fix released in 4.11.4

**Comment by burkmarr** [comment]
Thanks @elylucas.  I had plenty of toast for breakfast today!
